nucleus/base/models.py

```diff
@@ -19,26 +19,23 @@
 
 
 def send_instance_to_github(instance, branch=DEFAULT_BRANCH):
-    log.debug(f'send_instance_to_github, {instance._meta}, {instance.pk}')
+    log.debug(f'send_instance_to_github, {instance._meta.label_lower}, {instance.pk}')
     author = get_current_user()
-    # give it a few seconds to be sure the db
-    # transaction is committed
-    tasks.schedule_at('save_to_github',
-                      now() + timedelta(seconds=5),
-                      str(instance._meta),
-                      instance.pk,
-                      author.pk,
-                      branch)
+    tasks.schedule('nucleus:save_to_github',
+                   instance._meta.label_lower,  # app_name.model_name
+                   instance.pk,
+                   author.pk if author else None,
+                   branch)
 
 
-@receiver(post_save)
+@receiver(post_save, weak=False, dispatch_uid='send_to_github_signal')
```

Comment:
`weak=False` :+1: 

/me wonders why django defaults to weak=True that can result in signal not being triggered?

Reply:
same...